docs/yul.rst

```diff
@@ -559,8 +559,7 @@ regular strings in native encoding. For code,
 
 Grammar::
 
-    TopLevelObject = 'object' '{' Code? ( Object | Data )* '}'
-    Object = 'object' StringLiteral '{' Code? ( Object | Data )* '}'
+    Object = 'object' StringLiteral '{' Code ( Object | Data )* '}'
```

Comment:
```suggestion
    Object = 'object' StringLiteral '{' (Code | Object | Data )+ '}'
```

Reply:
I think an object has exactly one `Code`.

Reply:
But I don't think code is mandatory.

Reply:
This PR is suggesting that it should be :)